documentation/FAQ.md

```diff
@@ -18,10 +18,19 @@ Frequently Asked Questions
     Can not load or initialize libmscordaccore.so. The target runtime may not be initialized.
     ClrStack  failed
     ```
+    or
+    ```
+    Failed to load data access module, 0x80131c4f
+    You can run the debugger command 'setclrpath ' to control the load of libmscordaccore.so.
+    If that succeeds, the SOS command should work on retry.
+    For more information see https://go.microsoft.com/fwlink/?linkid=2135652
+    ```
     First try enabling the symbol downloading with `setsymbolserver -ms`. This is already enabled for `dotnet-dump analyze` and if SOS for lldb was installed with `dotnet-sos install`.
 
     If that doesn't work, try using the `setclrpath <directory>` command with a directory that contains the matching version of the DAC module. This is useful for private runtimes or debug builds that haven't been published to our symbol servers.
 
+    If this is a dump, the problem could also be that the dump is missing some memory required by SOS. Try generating a "full" dump (the default with `dotnet-dump collect` without a `--type` option) or add setting the crash dump generation (createdump) environment variable `COMPlus_DbgMiniDumpType=4`. For more details on crash dump generation see [here](https://github.com/dotnet/runtime/blob/master/docs/design/coreclr/botr/xplat-minidump-generation.md#configurationpolicy).
```

Comment:
nit: point this to the public docs https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dumps#collecting-dumps-on-crash That way we have less to change if we move this to the public docs.